secator/tasks/masscan.py

```
@@ -0,0 +1,180 @@
+	def on_cmd_done(self):
+		if not os.path.exists(self.output_path):
+			yield Error(message=f'Could not find JSON results in {self.output_path}')
+			return
+		yield Info(message=f'JSON results saved to {self.output_path}')
+		exclude_ports = self.get_opt_value('exclude_ports') or []
+		if exclude_ports:
+			exclude_ports = [int(p) for p in exclude_ports.split(',')]
+			yield Info(message=f'Excluded ports: {exclude_ports}')
+		port_count = 0
+		with open(self.output_path, 'r') as f:
+			for (ix, line) in enumerate(f):
```

Ruff (0.15.18)

[warning] Loop control variable `ix` not used within loop body

Rename unused `ix` to `_ix`

(B007)